tests/by-util/test_chown.rs

```diff
@@ -730,15 +730,16 @@ fn test_chown_file_notexisting() {
     let user_name = String::from(result.stdout_str().trim());
     assert!(!user_name.is_empty());
 
-    let _result = scene
+    let result = scene
         .ucmd()
-        .arg(user_name)
+        .arg(&user_name)
         .arg("--verbose")
         .arg("not_existing")
         .fails();
 
-    // TODO: uncomment once "failed to change ownership of '{}' to {}" added to stdout
-    // result.stderr_contains("retained as");
+    result.stdout_contains(format!(
+        "failed to change ownership of 'not_existing' to {user_name}"
```

Comment:
Interesting that this message is different from the one that was commented out. Was that just a mistake?

Reply:
You could also add this assertion to the rest of the expression:
```rust
scene
    .ucmd()
    ...
    .fails()
    .stdout_contains(...);
```

Reply:
The actual test tests a non existing file with verbose flag:
```
 chown -v 0 nf
```

The string "retained as" is printed by the command:
```
touch f
chown -v --from=42 43 f
```

These two cases came from the GNU test suite: tests/chown/basic.sh